packages/openapi3/src/schema-emitter.ts

```diff
@@ -837,9 +848,9 @@ export class OpenAPI3SchemaEmitter extends TypeEmitter<
       (p: Program, t: Type) => (getDeprecated(p, t) !== undefined ? true : undefined),
       "deprecated"
     );
-
-    this.#attachXmlObject(program, type as ModelProperty, schema);
-
+    if (!this.#isStdType(type)) {
```

Comment:
try `type.kind === 'ModelProperty'`?

Reply:
looks like the type can be `Model` as well.
```
@encodedName("application/xml", "XmlTag")
model Tag {
  name: string;
}
```

looks like
```
@xml.unwrapped
model Tag {
  name: string;
}
```
will add `wrapped: false` to `Tag`

Reply:
the same issue to `@Xml.attribute`

Reply:
not only `Model` and `ModelProperty`

like Scalar
```
 @name("ItemsName")
  scalar tag extends string;
```